pkg/cmd/reduce/main.go

```diff
@@ -114,21 +114,23 @@ func reduceSQL(path, contains string, workers int, verbose bool) (string, error)
 	}
 
 	interesting := func(ctx context.Context, f reduce.File) bool {
-		cmd := exec.CommandContext(ctx, path, "demo", "--empty")
-		// Disable telemetry and license generation.
-		cmd.Env = []string{"COCKROACH_SKIP_ENABLING_DIAGNOSTIC_REPORTING", "true"}
```

Comment:
You probably need to keep the env var. `--disable-demo-license` is not sufficient to disable telemetry and sentry crash reporting.


(unless you intend to keep telemetry)

Reply:
TIL